### ISSUE TITLE
Gauntlet incredibly slow since 0.7.7

### ISSUE THREAD

**Issue by Darxide23** [IAmPicard/StarTrekTimelinesSpreadsheet#126]
In 0.7.6 Gauntlet was snappy and quick. Once updated to 0.7.7 there is a significant delay when doing any Gauntlet activity. Gauntlet is one of the primary reasons I use this tool (the other is for Voyage calculations) so losing the quickness of completing Gauntlet is a real detriment.

**Comment by Darxide23** [comment]
Just checked them side by side. 0.7.6 is much, much faster in Gauntlet than 0.7.7.

**Comment by Darxide23** [comment]
Alright, so this isn't just a Gauntlet thing. 0.7.7 is much slower all around. I did some video caps to illustrate this. I also made sure to uninstall and reinstall 0.7.7 just in case and that didn't help any at all.

https://www.youtube.com/watch?v=jS4QfeyC26Y

**Comment by IAmPicard** [comment]
There's no arguing with that, wow. I suspect this is related to upgrading electron to 3.0, this brought along a new version of chromium (from 61 to 66), which is the same engine that powers Chrome.

It's weird that I don't see this slowdown on my machines but perhaps it's somehow related to a particular config. FWIW, I'm running updated Windows 10 on current or previous generation Intel CPUs.

What is your machine config?

Are you using Chrome as your browser? Did you notice a slowdown in Chrome sometime in the past 3 months?

For others reading this thread, please report your observations as well.

I could go back to electron 2.0, but there are a lot of nice things in the newer ones, including significant RAM usage reduction, as well as security upgrades and so on.

**Comment by pim69** [comment]
I created a github account specifically to also add my experience with this issue.  I am running Windows 7 64bit Ultimate SP1 fully updated, core i52500k cpu, 8gb RAM.  Just logging on is taking DRAMATICALLY longer than it did on the prior version for me.  Before this update, I had no performance problems whatsoever.  

In case this helps, I have noticed two specific places it hangs during login:
1. caching images (always stops on the second last image, like 455/456 images for example)
2. if I cancel and reload the game impatient waiting for 1. to finish, it instead hangs on "Finishing up..." for a couple of minutes.

I have noted no performance difference on Chrome browser lately.  In fact, I recently changed from a standard HDD to a solid state drive, so otherwise my system performance is dramatically improved.  the SSD change was a couple of months ago, this problem only began for me with version 0.7.7.

Thanks!

**Comment by IAmPicard** [comment]
I found this [interesting bug](https://github.com/electron/electron/issues/13829) in electron 3.0 which might explain the problem.

I'm still confused about why I'm not seeing the issue myself; I suspect it is config-specific.

In case you were curious to try Windows 10 on your machine, maybe give that a try? FWIW, I believe Windows 7 has been out of support for ~4 years.

**Comment by dewittified** [comment]
I don't have any speed issues. Win10

**Comment by Lombra** [comment]
Hi,

As I mentioned in my ticket, my VirtualBox network interface definitely seemed to be the cause of this. Do you guys have any network interfaces installed besides your primary one? Try disabling them and see if it makes a difference. I'm sure the Electron people would love to know, assuming this is actually related to network requests like it was in my case.

(also, Windows 7 is supported until 2020!)

**Comment by Darxide23** [comment]
I'm on Win7 SP1. It's not a slow machine at all. I built it as a medium-high end gaming right about 5 years ago, so it's due for an overhaul by now, but no slouch by any means. Looking at others it seems the issue might be between Win7 and Win10? I have no systems running Win10 and I don't foresee that changing in the near future.

**Comment by pim69** [comment]
Wanted to follow up that performance improved for me after a while using the same version 0.7.7.  Although I have Cisco AnyConnect Secure Mobility client VPN for work, having it enabled or shut down made no difference.

However, I realized there were some optional Windows 7 updates I had not yet applied with random software compatibility fixes, so I installed ALL of those.  The problem did not go away immediately, but it seemed to be soon after that the performance improved very noticeably.  My guess is one of those likely helped me, because otherwise I have no idea what made a difference.
